### DISCUSSION TITLE
Why am I getting different results than docs?

### DISCUSSION THREAD

**Discussion by leonstafford** [pestphp/pest#266] [Q&A]
In the [Underlying Test Case: Overview](https://pestphp.com/docs/underlying-test-case#overview), it says the type of `$this` at that point should default to `\PHPUnit\Framework\TestCase`, but I receive `P\Tests\Unit\AuditorTest` (my test class's name). Is this anything to do with my setup or something to update on docs?

(no Issues available for the [docs repo](https://github.com/pestphp/pestphp.com) so asking in here)

My [example project](https://github.com/leonstafford/WhatWouldViktorDo) where I noticed this.

Original chunk from docs:

```php
<?php

it('has home', function () {
    $this->assertTrue(true);

    // \PHPUnit\Framework\TestCase
    echo get_class($this);
});
```

**Comment by owenvoke** [comment]
I think what it's meant to be implying is that `$this` extends `PHPUnit\Framework\TestCase` by default.

Perhaps the use of `get_class()` is a bit confusing, and `get_parent_class()` would be more realistic. 🤷🏻

**Reply by leonstafford** [reply]
Thanks @owenvoke, that makes sense!
